Samples/Windowing/README.md

```diff
@@ -1,9 +1,14 @@
 ---
-title: "AppWindow gallery sample" 
+name: "AppWindow gallery sample" 
```

Comment:
@samanthamsong should AppWindow be changed to Windowing? Both here and in other sections of the ReadMe below?